functions/go/apply-setters/README.md

```diff
@@ -0,0 +1,94 @@
+# apply-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Apply setter values on resources fields. May set either the complete or partial field value.
+
+<!--mdtogo-->
+
+Setters provide a solution for template-free setting of field values. They are a
+safer alternative to other substitution techniques which do not have the context
+of the structured data. Setters may be invoked to modify the configuration
+using `apply-setters` function to set values.
+
+### Synopsis
```

Comment:
Where do you discuss:
- scalar vs list
- interpolation

Keep in mind, this is the comprehensive reference documentation for setters.

Reply:
Separated scalar vs array values setting. Consumers need not know about interpolation. It is not mentioned even in the error messages. Added example for setting partial field value.